File: Fibonacci/python/fibonacci.py
#!/usr/bin/env python3
def fibonacci(n):
    if n < 2:
        return 1
    return fibonacci(n-2) + fibonacci(n-1)

numero=int(input("Calcolo n-esimo numero di Fibonacci: "))
print("F(%d)=%d" % (numero, fibonacci(numero)))

File: Fibonacci/ruby/fibonacci.rb
#!/usr/bin/ruby

def fib(x)
  x < 2 ? 1 : fib(x-1) + fib(x-2)
end

print "Calcolo n-esimo numero di Fibonacci: "
numero  = gets.chomp.to_i
puts "F(#{numero})=#{fib(numero)}"

